spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -312,6 +312,7 @@ private void registerAggregateBeanDefinitions(Configurer configurer, BeanDefinit
                                                        () -> beanFactory.getBean(EntityManagerProvider.class)),
                                         aggregateType,
                                         c.eventBus(),
+                                        c::repository,
```

Comment:
If `c::repository` is the default lambda used as the `RepositoryProvider` lambda, maybe it's nice to introduce a static function for it, like we've done in the `ConsistentHashChangeListener` interface.

Reply:
I don't think it can be done, since `Configuration#repository` is non-static method which cannot be referenced from static context. Or am I missing your idea?

Reply:
Ok...think I agree here. This is what it would look like adding such a function to the `RepositoryProvider` interface:
```
static RepositoryProvider defaultRepositoryProvider(Configuration conf) {
    return conf::repository;
}
```
Don't really like wiring in the `Configuration` there :/